src/api/Metadata.js

```diff
@@ -368,6 +459,7 @@ class Metadata extends File {
         errorCallback: ElementsErrorCallback,
```

Comment:
Should we need to update the successCallback type to include template instances now?